lib/models/obms.js

```diff
@@ -106,6 +108,37 @@ function ObmsModelFactory (
             next();
         },
 
+        publishEvent: function(nodeId, next) {
+            return waterline.nodes.findByIdentifier(nodeId)
```

Comment:
needByIdnetifier will throw Errors.NotFoundError,  not intended to let it throw error